### ISSUE TITLE
Only the last file in a zip file is being ingested

### ISSUE THREAD

**Issue by tomwrobel** [tomwrobel/ora_data_model#161]
The problem:

If a zip file is uploaded with multiple files, only the last file in the zip is actually ingested into Hyrax via Sword.

It appears that the zip file is being unzipped, and one file (the last?) taken from it and ingested

Test UUID on ora4-qa-review: uuid_0c24daa7-f6eb-4311-b659-51468320781f

Notes:

For a 'normal' file deposit, the following set of temp files are created:

```
-- tmp/{hash}/
-- tmp/{hash}/{file}
-- tmp/{hash}/contents
-- tmp/{hash}/contents/{file}
-- tmp/{hash}/bag
-- tmp/{hash}/bag/bag-info.txt
-- tmp/{hash}/bag/bagit.txt
-- tmp/{hash}/bag/manifest-md5.txt
-- tmp/{hash}/bag/manifest-sha1.txt
-- tmp/{hash}/bag/tagmanifest-md5.txt
-- tmp/{hash}/bag/tagmanifest-sha1.txt
-- tmp/{hash}/bag/data
-- tmp/{hash}/bag/data/{file}
```

However, if a zip file is deposited, the following set of files are created:

for the file 'kitties.zip', containing pic1.jpg, pic2.jpg, pic3.jpg

```
-- tmp/{hash}/
-- tmp/{hash}/kitties.zip
-- tmp/{hash}/contents
-- tmp/{hash}/contents/pic1.jpg
-- tmp/{hash}/contents/pic2.jpg
-- tmp/{hash}/contents/pic3.jpg
-- tmp/{hash}/bag
-- tmp/{hash}/bag/bag-info.txt
-- tmp/{hash}/bag/bagit.txt
-- tmp/{hash}/bag/manifest-md5.txt
-- tmp/{hash}/bag/manifest-sha1.txt
-- tmp/{hash}/bag/tagmanifest-md5.txt
-- tmp/{hash}/bag/tagmanifest-sha1.txt
-- tmp/{hash}/bag/data
-- tmp/{hash}bag/data/pic1.jpg
-- tmp/{hash}bag/data/pic2.jpg
-- tmp/{hash}bag/data/pic3.jpg
```

Only pic3.jpg is attached to the fileset for download via Hyrax. The 'original_file.original_name' for that file is 'pic3.jpg', although the fileset will have the file_name set by Sword2 which will be the name of the zip file.

Linked ticket: https://trello.com/c/N8prMnWR

**Comment by tomwrobel** [comment]
@eugeniobarrio @jjpartridge @mrdsaunders what is the priority for this issue?

It presents a relatively serious potential problem. Easily solved by not allowing zip file deposit, but we'll need to work on it nevertheless

**Comment by tomwrobel** [comment]
Solved in [feature/ora_customizations 4f48655] Don't unpack zip files if config is set

by using a configuration option

@lintuula are you able to redeposit kitties.zip in order to test?

**Comment by lintuula** [comment]
Tested and now downloads the whole zip file.
